### PR TITLE
2625: Preventing Key Bindings when using the Accessibility Window

### LINKED ISSUE #2625: "Enter," Chat Window Shortcut Conflicts with Accessibility SUpport
### Environment
What version of MegaMek does your issue apply to?  Megamek version 0.47.16, stand-alone.
What operating system are you using?  Mac.
What java version are you using? Java/OpenJDK version 1.8.0_282.

### Description
The default shortcut key used to open the chat window, "Enter," conflicts with the Accessibility window used to control the game with a screen reader. 

Steps:
1. Launch Megamek.
2. Open a scenario.
3. Open the accessibility window from the View menu.
4. Try to type a command, such as #help or #tile. Press Enter to submit.

Actual results:
The focus jumps from the Accessibility window, and the command is not submitted. The chat window presumably opens.

Expected Results:
Focus should remain in the command box. The command should be submitted. The chat window should remain closed.

Possible Resolutions:
Change the default chat key shortcut to something less frequently used, such as the grave accent. Long-term, perhaps work on making the chat window accessible and integrating the separate "accessibility window," functionality into it.

### Files
NA

### PR DESCRIPTION
**Pull request by Windchild292**
This fixes #2625 by preventing key bindings when using the accessibility window. The code change for the fix is commented.

### CODE REVIEW

File: megamek/src/megamek/client/ui/swing/ClientGUI.java [hunk 1]
```diff
@@ -2472,18 +2442,21 @@ public void unitSelected(BoardViewEvent b) {
...
         return ignoreHotKeys 
-                || (gameOptionsDialog != null && gameOptionsDialog.isVisible())
-                || (about != null && about.isVisible()) 
-                || (help != null && help.isVisible()) 
-                || (setdlg != null && setdlg.isVisible());
+                || ((gameOptionsDialog != null) && gameOptionsDialog.isVisible())
+                || ((about != null) && about.isVisible())
+                || ((help != null) && help.isVisible())
+                || ((setdlg != null) && setdlg.isVisible())
+                || ((aw != null) && aw.isVisible());
```
Comment (Windchild292) [inline megamek/src/megamek/client/ui/swing/ClientGUI.java:2449]: The final fix

**Comment by NickAragua** [comment]
Can you point out the actual code changes in this PR? It's full of random noise consisting of irrelevant variable name and formatting changes.

**Comment by Windchild292** [comment]
I did point it out?

**Comment by NickAragua** [comment]
Ok, as long as that's the only meaningful change.

**Review by NickAragua — APPROVED** [review]

**Pull request merged**